serial.c

```diff
@@ -145,21 +152,44 @@ static ssize_t serial_write_data(struct iiod_client_pdata *io_data,
 	return ret;
 }
 
+void sleep_one_ms(void)
+{
+#ifdef _WIN32
+	Sleep(1);
+#else
+	const struct timespec ts = {
+		/* One millisecond */
+		.tv_nsec = 1 * 1000 * 1000,
+	};
+
+	nanosleep(&ts, NULL);
+#endif
+}
+
```

Comment:
should this be in utils.c rather than here? (I guess it's a single use case?)

Reply:
utils.c is for the high-level API, we'd have to export it as a public symbol (as the serial backend can be a module) and I prefer to avoid that as much as possible.

Reply:
makes sense.